src/app.config.js

```diff
@@ -9,6 +9,7 @@ const Config = {
     iotHubManager: `${baseUrl}/iothubmanager/v1/`,
     telemetry: `${baseUrl}/telemetry/v1/`,
     deviceSimulation: `${baseUrl}/devicesimulation/v1/`,
+    diagnosticsApiUrl: `${baseUrl}/diagnostics/v1/`,
```

Comment:
Remove "ApiUrl" from this name. Be consistent with the others.

Reply:
@isaac73 this needs to be addressed

Reply:
done